CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -133,6 +143,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 			let pageOffset = CGFloat(index) * pageWidth - collectionView.contentInset.top
 			proposedContentOffset = CGPoint(x: collectionView.contentOffset.x, y: pageOffset)
 			shouldAnimate = abs(collectionView.contentOffset.y - pageOffset) > 1 ? animated : false
+            
+        default:
+            proposedContentOffset = .zero
+            shouldAnimate = false
```

Comment:
The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?